### PR TITLE
✨ feat: featbit light version

### PR DESCRIPTION
**Pull request by deleteLater**
### light version
- no kafka
- no clickhouse
- use **redis pub/sub** to handle flag and segment change messages: **api pub, evaluation-server sub**
- use **redis list** to handle end-user and insight messages: **evaluation server RPush, api LPOP**
- the Redis cache is now maintained by the API server instead of the evaluation server

### build and run locally
```bash
cd featbit
docker compose -f docker-compose-dev.yml up -d
```

### docs
- [x] README.md
- [x] [docs.featbit.co](http://docs.featbit.co/)

### remarks
- updated doc: https://docs.featbit.co/docs/installation/full-installation
- new doc: https://docs.featbit.co/docs/installation/faq
- new doc: https://docs.featbit.co/docs/tech-stack/starter-vs.-professional

### CODE REVIEW

**Comment by deleteLater** [comment]
change `LIGHT_VERSION` to `IS_PRO`

File: modules/data-analytics/.vscode/settings.json
Comment (cosmos-explorer) [inline modules/data-analytics/.vscode/settings.json:1]: This file should not be committed

File: modules/back-end/src/Api/appsettings.json [hunk 1]
```diff
@@ -20,5 +20,9 @@
   "OLAP": {
     "ServiceHost": "http://da-server"
   },
-  "AllowedHosts": "*"
+  "Redis": {
+    "ConnectionString": "redis:6379"
+  },
+  "AllowedHosts": "*",
+  "LIGHT_VERSION": true
```
Comment (cosmos-explorer) [inline modules/back-end/src/Api/appsettings.json]: "IS_PRO": false

Comment (deleteLater) [reply modules/back-end/src/Api/appsettings.json]: fixed in [070bb5e](https://github.com/featbit/featbit/pull/313/commits/070bb5ee9b1470bda4ec576142ff563b9563dce2)

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -64,10 +64,12 @@ services:
     environment:
       REDIS_HOST: redis
       REDIS_PORT: 6379
-      LIGHT_VERSION: true
       MONGO_URI: mongodb://admin:password@mongodb:27017
       MONGO_INITDB_DATABASE: featbit
       MONGO_HOST: mongodb
+      CLICKHOUSE_HOST: clickhouse-server
+      CLICKHOUSE_KAFKA_HOSTS: kafka:9092
+      IS_PRO: false
```
Comment (deleteLater) [inline docker-compose-dev.yml]: Why we need clickhouse in lite version

**Review by cosmos-explorer — APPROVED** [review]

**Pull request merged**